apps/financial-aid/web-veita/src/components/MunicipalityAdminSettings/MunicipalityNumberInput/MunicipalityNumberInput.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box } from '@island.is/island-ui/core'
+import { NumberInput } from '@island.is/financial-aid-web/veita/src/components'
+import { AidName, KeyMapping } from '@island.is/financial-aid/shared/lib'
+
+interface Props {
+  id: string
+  aid: number
+  prefix: string
+  hasAidError: boolean
+  updateHandler: (value: number) => void
+}
+
+const MunicipalityAdminInput = ({
+  id,
+  aid,
+  prefix,
+  hasAidError,
+  updateHandler,
+}: Props) => {
+  const label: KeyMapping<string, string> = {
+    [AidName.OWNPLACE]: 'Eigið húsnæði',
+    [AidName.REGISTEREDRENTING]: 'Leiga með þinglýstan leigusamning',
+    [AidName.UNREGISTEREDRENTING]: 'Leiga með óþinglýstan leigusamning',
+    [AidName.WITHOTHERS]: 'Býr eða leigir hjá öðrum án þinglýsts leigusamnings',
+    [AidName.LIVESWITHPARENTS]: 'Býr hjá foreldrum',
+    [AidName.UNKNOWN]: 'Annað',
```

Comment:
Why is this not in shared? and then we dont have to make a special component just for this and just use NumberInput and Box in the parent component?

Reply:
I feel like it is ok to keep the input in a separate component because things are getting a bit clustered and nested in the parent component:
![image](https://user-images.githubusercontent.com/43727721/154303613-7e4a9d7b-32a8-49b6-96f3-5b55039c2840.png)